redaxo/src/core/lib/console/application.php

```diff
@@ -39,14 +39,16 @@ protected function doRunCommand(Command $command, InputInterface $input, OutputI
 
     private function loadPackages(rex_console_command $command)
     {
-        if ('ydeploy:migrate' !== $command->getName()) {
-            require rex_path::core('packages.php');
+        if ('ydeploy:migrate' === $command->getName()) {
+            $command->getPackage()->boot();
 
             return;
         }
 
-        $package = $command->getPackage();
-        $package->enlist();
-        $package->boot();
+        foreach (rex::getConfig('package-order') as $packageId) {
```

Comment:
finde ich nicht blöd.

ggf. könnte man noch abwägen ob man dieses foreach + den EP in eine methode `rex_package::bootPackages()` auslagert?